app/controllers/completed_applications_controller.rb

```diff
@@ -39,8 +36,13 @@ def recreate
   end
 
   def create_pse
-    # TODO: implement functionality
-    redirect_to completed_crime_application_path(crime_application: current_crime_application)
+    pse_application = Pse::FindOrCreate.new(
```

Comment:
Small think thinking we could have a PseApplicationController subclass from CompletedApplicationController and then have `create` rather than `create_pse` so that if we need other actions its more RESTful

But I think this is fine really and if we start to need more actions we could refactor

Reply:
Agreed. Let's review when we add the next type.